english/crochet/crochet.md

```diff
@@ -0,0 +1,3 @@
+Crochet teddy bears are the cutest.  You should start working on one today.  
+
+Click here to [return to reality] (https://github.com/udacity/create-your-own-adventure/blob/master/english/marshmallow.md).
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.